vendor/gitea.com/macaron/session/postgres/postgres.go

```diff
@@ -121,18 +121,20 @@ func (p *PostgresProvider) Init(maxlifetime int64, connStr string) (err error) {
 
 // Read returns raw session store by session ID.
 func (p *PostgresProvider) Read(sid string) (session.RawStore, error) {
+	now := time.Now().Unix()
 	var data []byte
-	err := p.c.QueryRow("SELECT data FROM session WHERE key=$1", sid).Scan(&data)
+	expiry := now
+	err := p.c.QueryRow("SELECT data FROM session WHERE key=$1", sid).Scan(&data, &expiry)
```

Comment:
DAMN! This needs to be:

```suggestion
	err := p.c.QueryRow("SELECT data, expiry FROM session WHERE key=$1", sid).Scan(&data, &expiry)
```

Reply:
https://gitea.com/macaron/session/pulls/12 I've used 🧙 powers to merge the linked PR, you should be able to use a go get -u on sessions and a make vendor

Reply:
I'm having difficulty getting go get -u to update it...

Reply:
done